sport/app/cricket/feed/cricketPaFeed.scala

```diff
@@ -4,17 +4,18 @@ import akka.actor.ActorSystem
 import akka.stream.Materializer
 import common.GuLogging
 import cricket.feed.CricketThrottler
-import org.joda.time.{DateTime, LocalDate}
-import org.joda.time.format.{DateTimeFormat, DateTimeFormatter}
+import java.time.LocalDate
 import play.api.libs.ws.WSClient
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.xml.XML
 
+import java.text.SimpleDateFormat
+
 case class CricketFeedException(message: String) extends RuntimeException(message)
 
 object PaFeed {
-  val dateFormat: DateTimeFormatter = DateTimeFormat.forPattern("yyyy-MM-dd")
+  val dateFormat = new SimpleDateFormat("yyyy-MM-dd") // Note that there is an implicit timezone assumption here
```

Comment:
Can we make the timezone explicit at least for clarity?

Reply:
Done. I originally resisted because I didn't want to change behaviour, but on a date formatting this will not have any adverse side effect :)